examples/campaign_management/validate_text_ad.py

```diff
@@ -0,0 +1,98 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example shows use of the validateOnly header for an expanded text ad.
+
+No objects will be created, but exceptions will still be thrown.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, ad_group_id):
+    ad_group_ad_operation = client.get_type('AdGroupAdOperation', version='v2')
+    ad_group_ad = ad_group_ad_operation.create
+    ad_group_service = client.get_service('AdGroupService', version='v2')
+    ad_group_ad.ad_group.value = ad_group_service.ad_group_path(customer_id,
+        ad_group_id)
+    ad_group_ad.status = client.get_type('AdGroupAdStatusEnum',
+        version='v2').PAUSED
+    ad_group_ad.ad.expanded_text_ad.description.value = 'Luxury Cruise to Mars'
+    ad_group_ad.ad.expanded_text_ad.headline_part1.value = (
+        'Visit the Red Planet in style.')
+    ad_group_ad.ad.expanded_text_ad.headline_part2.value = (
+        'Low-gravity fun for everyone!!')
+    final_url = ad_group_ad.ad.final_urls.add()
+    final_url.value = 'http://www.example.com/'
+
+    ad_group_ad_service = client.get_service('AdGroupAdService', version='v2')
+    # Attempt the mutate with validate_only=True
+    try:
+        response = ad_group_ad_service.mutate_ad_group_ads(customer_id,
+            [ad_group_ad_operation], partial_failure=False, validate_only=True)
+        print('"Expanded text ad validated successfully.')
+    except GoogleAdsException as ex:
+        # This will be hit if there is a validation error from the server.
+        print(f'Request with ID "{ex.request_id}" failed with status '
+              f'"{ex.error.code().name}".')
+        print('There may have been validation error(s) while adding expanded '
+              'text ad.')
+        for error in ex.failure.errors:
+            # Note: Depending on the ad type, you may get back policy violation
+            # errors as either PolicyFindingError or PolicyViolationError.
+            # ExpandedTextAds return errors as PolicyFindingError, so only this
+            # case is illustrated here. For additional details, see
+            # https://developers.google.com/google-ads/api/docs/policy-exemption/overview
+            if (error.error_code.policy_finding_error ==
+                client.get_type('PolicyFindingErrorEnum',
+                                version='v2').POLICY_FINDING):
+                if error.details.policy_finding_details:
+                    count = 1
+                    details = (error.details.policy_finding_details
+                               .policy_topic_entries)
+                    for entry in details:
+                        print(f'{count}) Policy topic entry with topic '
+                              f'{entry.topic} and type {entry} '
```

Comment:
Please wrap entry topic and type with a quote too.

Reply:
fixed.

Reply:
I did this but it looks weird because of the newlines in the entries:

```
1) Policy topic entry with topic "value: "SYMBOLS"
" and type "topic {
  value: "SYMBOLS"
}
type: PROHIBITED
evidences {
  text_list {
    texts {
      value: "!\n"
    }
  }
}
evidences {
  text_list {
    texts {
      value: "everyone!!"
    }
  }
}
" was found.
```

Reply:
I fixed the print to be `type="{entry.type}" ` and it still looks weird. 

```
1) Policy topic entry with topic="value: "SYMBOLS"
" and type="2" was found.
```

Do you recommend: (a) mapping entry.type to the appropriate enum name or (b) just printing the whole `entry` without quotes. My vote is for (b) as the enums get mapped to names anyway. (I think the golden could use an update about this).

Reply:
I'm actually confused by this printing too.
Why do we want to print both entry and entry.topic?
entry.topic should be a part of entry?

So, why don't we just print {entry} (in the next line)?

WDYT?

Reply:
Much better. Let's go with `print(f'{count}) Policy topic entry: \n{entry}\n')`:

```
1) Policy topic entry: 
topic {
  value: "SYMBOLS"
}
type: PROHIBITED
evidences {
  text_list {
    texts {
      value: "!\n"
    }
  }
}
evidences {
  text_list {
    texts {
      value: "everyone!!"
    }
  }
}

```

Reply:
Great